modules/phase0/src/main/resources/phase0/Phase0_Instrument_Matrix - Spectroscopy.tsv

```diff
@@ -379,41 +379,41 @@ GMOS-S	B600 IFU2 NS	ifu	IFU-NS-2	5	7	B600	g	0.398	0.552	0.461	0.154	2723	no	Nod&
-FLAMINGOS2	R3K + Ks + 0.36"	singleslit,multislit	3pixel	0.540	263	R3000	K-short	1.991	2.32	2.1555	0.329	1933	no		s
-FLAMINGOS2	R3K + Kb + 0.36"	singleslit,multislit	3pixel	0.540	263	R3000	K-blue	1.934	2.177	2.0555	0.243	1933	no		s
-FLAMINGOS2	R3K + Kr + 0.36"	singleslit,multislit	3pixel	0.540	263	R3000	K-red	2.181	2.446	2.3135	0.265	1933	no		s
-FLAMINGOS2	JH 0.36"	singleslit,multislit	3pixel	0.540	263	R1200JH	JH	0.97	1.805	1.3875	0.835	600	no		s
-FLAMINGOS2	HK 0.36"	singleslit,multislit	3pixel	0.540	263	R1200HK	HK	1.261	2.551	1.906	1.29	600	no		s
-FLAMINGOS2	R3K + J + 0.36"	singleslit,multislit	4pixel	0.720	263	R3000	J	1.175	1.333	1.254	0.158	1400	no		s
-FLAMINGOS2	R3K + H + 0.36"	singleslit,multislit	4pixel	0.720	263	R3000	H	1.486	1.775	1.6305	0.289	1400	no		s
-FLAMINGOS2	R3K + Ks + 0.36"	singleslit,multislit	4pixel	0.720	263	R3000	K-short	1.991	2.32	2.1555	0.329	1450	no		s
-FLAMINGOS2	R3K + Kb + 0.36"	singleslit,multislit	4pixel	0.720	263	R3000	K-blue	1.934	2.177	2.0555	0.243	1450	no		s
-FLAMINGOS2	R3K + Kr + 0.36"	singleslit,multislit	4pixel	0.720	263	R3000	K-red	2.181	2.446	2.3135	0.265	1450	no		s
-FLAMINGOS2	JH 0.36"	singleslit,multislit	4pixel	0.720	263	R1200JH	JH	0.97	1.805	1.3875	0.835	450	no		s
-FLAMINGOS2	HK 0.36"	singleslit,multislit	4pixel	0.720	263	R1200HK	HK	1.261	2.551	1.906	1.29	450	no		s
-FLAMINGOS2	R3K + J + 0.36"	singleslit,multislit	6pixel	1.080	263	R3000	J	1.175	1.333	1.254	0.158	933	no		s
-FLAMINGOS2	R3K + H + 0.36"	singleslit,multislit	6pixel	1.080	263	R3000	H	1.486	1.775	1.6305	0.289	933	no		s
-FLAMINGOS2	R3K + Ks + 0.36"	singleslit,multislit	6pixel	1.080	263	R3000	K-short	1.991	2.32	2.1555	0.329	967	no		s
-FLAMINGOS2	R3K + Kb + 0.36"	singleslit,multislit	6pixel	1.080	263	R3000	K-blue	1.934	2.177	2.0555	0.243	967	no		s
-FLAMINGOS2	R3K + Kr + 0.36"	singleslit,multislit	6pixel	1.080	263	R3000	K-red	2.181	2.446	2.3135	0.265	967	no		s
-FLAMINGOS2	JH 0.36"	singleslit,multislit	6pixel	1.080	263	R1200JH	JH	0.97	1.805	1.3875	0.835	300	no		s
-FLAMINGOS2	HK 0.36"	singleslit,multislit	6pixel	1.080	263	R1200HK	HK	1.261	2.551	1.906	1.29	300	no		s
-FLAMINGOS2	R3K + J + 0.36"	singleslit,multislit	8pixel	1.440	263	R3000	J	1.175	1.333	1.254	0.158	700	no		s
-FLAMINGOS2	R3K + H + 0.36"	singleslit,multislit	8pixel	1.440	263	R3000	H	1.486	1.775	1.6305	0.289	700	no		s
-FLAMINGOS2	R3K + Ks + 0.36"	singleslit,multislit	8pixel	1.440	263	R3000	K-short	1.991	2.32	2.1555	0.329	725	no		s
-FLAMINGOS2	R3K + Kb + 0.36"	singleslit,multislit	8pixel	1.440	263	R3000	K-blue	1.934	2.177	2.0555	0.243	725	no		s
-FLAMINGOS2	R3K + Kr + 0.36"	singleslit,multislit	8pixel	1.440	263	R3000	K-red	2.181	2.446	2.3135	0.265	725	no		s
-FLAMINGOS2	JH 0.36"	singleslit,multislit	8pixel	1.440	263	R1200JH	JH	0.97	1.805	1.3875	0.835	225	no		s
-FLAMINGOS2	HK 0.36"	singleslit,multislit	8pixel	1.440	263	R1200HK	HK	1.261	2.551	1.906	1.29	225	no		s
+Flamingos2	R3K + J + 0.36"	singleslit,multislit	Long Slit 2px	0.360	263	R3000	J	1.175	1.333	1.254	0.158	2800	no		s

```

Comment:
The `.tsv` file is maintained by science, as I understand it.  The idea being that we can just replace this with the latest version from them. Were these changes initiated by them?  If not, we should probably convince them to update their version accordingly or else parse their values as is.

Reply:
Yes, it was taken from the latest google sheet with the modes which is being maintained by science